pkgs/applications/networking/onionshare/default.nix

```diff
@@ -7,24 +7,28 @@
   isPy3k,
   flask,
   flask-httpauth,
+  flask-socketio,
   stem,
+  psutil,
   pyqt5,
   pycrypto,
+  pyside2,
   pysocks,
   pytest,
+  qrcode,
   qt5,
   requests,
   tor,
   obfs4,
 }:
 
 let
```

Comment:
Please remove this let in. It no longer servers any purpose.

Reply:
It still serves a purpose: both derivations use `src`